package/Dockerfile

```diff
@@ -11,12 +11,12 @@ RUN mkdir /root/.kube && \
 WORKDIR /var/lib/rancher
 
 ENV CATTLE_HELM_VERSION v2.10.0-rancher9
-ENV CATTLE_MACHINE_VERSION v0.15.0-rancher5-2
+ENV CATTLE_MACHINE_VERSION v0.15.0-rancher6-1
 ENV LOGLEVEL_VERSION v0.1.1
 ENV TINI_VERSION v0.18.0
 ENV TELEMETRY_VERSION v0.5.2
 
-RUN curl -sLf https://github.com/rancher/machine-package/releases/download/${CATTLE_MACHINE_VERSION}/docker-machine.tar.gz | tar xvzf - -C /usr/bin && \
+RUN curl -sLf https://github.com/rancher/machine-package/releases/download/${CATTLE_MACHINE_VERSION}/docker-machine-amd64.tar.gz | tar xvzf - -C /usr/bin && \
```

Comment:
putting amd64 for now as a full multi cluster app support is coming with https://github.com/rancher/rancher/pull/18574/files. We can change to the latest machine package release there too.